targets/TARGET_Cypress/TARGET_PSOC6/TARGET_CY8CKIT_062_WIFI_BT/GeneratedSource/cycfg_qspi_memslot.c

```diff
@@ -24,7 +24,7 @@
 
 #include "cycfg_qspi_memslot.h"
```

Comment:
shouldn't similar changes be made to other boards too?

Reply:
I'm hoping your upcoming QSPI PR will take care of other boards. QSPI isn't working with this PR alone. I had to make these changes to get QSPI to build first.